### PR TITLE
Splits and renames the "Bank & insurances" category

### PR DESCRIPTION
**Pull request by caugner**
In Germany, only banks are allowed to open, which excludes moneylenders and insurance offices.

Therefore I suggest splitting the category into three subcategories and give the category a neutral name.

### CODE REVIEW

File: locales/en.json [hunk 1]
```diff
@@ -61,7 +61,7 @@
   },
   "filter": {
     "bakery": "Bakeries",
-    "bank": "Banks & insurance companies",
+    "bank": "Financial servies",
```
Comment (francois2metz) [inline locales/en.json:64]: services

Comment (caugner) [reply locales/en.json:64]: Fixed in 5492851.

**Comment by caugner** [comment]
@francois2metz This is ready for review/merge now.

**Comment by francois2metz** [comment]
Merged in b5e0c07357a696253576bf4eebbabe40b84e5060. Thanks :green_heart: :purple_heart:

**Pull request closed without merging**